nautobot/utilities/tests/test_utils.py

```diff
@@ -231,6 +233,10 @@ def test_get_route_for_model(self):
         self.assertEqual(get_route_for_model(Device, "list", api=True), "dcim-api:device-list")
         self.assertEqual(get_route_for_model("dcim.site", "detail", api=True), "dcim-api:site-detail")
         self.assertEqual(get_route_for_model(Site, "detail", api=True), "dcim-api:site-detail")
+        self.assertEqual(get_route_for_model(ContentType, "list", api=True), "extras-api:contenttype-list")
+        self.assertEqual(get_route_for_model(ContentType, "detail", api=True), "extras-api:contenttype-detail")
+        self.assertEqual(get_route_for_model(Group, "list", api=True), "users-api:group-list")
+        self.assertEqual(get_route_for_model(Group, "detail", api=True), "users-api:group-detail")
```

Comment:
Hmm, I suppose for the `api=False` case we don't actually have viable routes for either of these models, do we? I guess it's inline with how `get_route_for_model` currently works that it just tries to guess the route name, but doesn't actually verify that it's valid though. Ah well.